src/appleseed-max-impl/appleseedrenderer/tilecallback.cpp

```diff
@@ -158,9 +157,19 @@ void TileCallback::on_tile_begin(
     const size_t x = tile_x * props.m_tile_width;
     const size_t y = tile_y * props.m_tile_height;
 
+    // Choose one color per thread (see https://www.shadertoy.com/view/wlKXDm).
```

Comment:
Could we make a function to compute that color, shared across all apps and plugins? It could be defined in `foundation/image/colormap.h`, for instance.

Reply:
Yep, that is possible. Saves some code in the plugins.